### ISSUE TITLE
Update raw SQL documentation (interpolated and string)

### ISSUE THREAD

**Issue by divega** [aspnet/EntityFramework.Docs#410]
This now produces a parameterized query:

``` C#
var namePattern = "M%T%";
var matchingPeople= context.People
    .FromSql($"SELECT p.* FROM People AS p WHERE p.Name LIKE {namePattern}");
```

**Comment by divega** [comment]
Superseded by https://github.com/aspnet/EntityFramework.Docs/issues/447.

**Comment by divega** [comment]
Not superseded any more. We need individual issues to assign them.

**Comment by anpete** [comment]
@divega Should I wait on this in case we make changes?

**Comment by roji** [comment]
This was done in #1673, closing.
